docs/updating.md

```diff
@@ -61,7 +61,18 @@ setting you use.
 
 ## Preventing Commit of Merge Conflicts
 
-If you use `--conflict rej` (the default):
+If you use `--conflict inline` (the default) then you need to check for conflicts marks in your files: 
```

Comment:
* `marks` ->  `markers`
* An empty line is missing.
* The line length exceeds 88 characters, so [Prettier won't be happy](https://github.com/copier-org/copier/blob/5fa3b9fb772101ac67476ab235b8e66ca26900de/.prettierrc.yml#L1).

```suggestion
If you use `--conflict inline` (the default) then you need to check for conflicts
markers in your files:

```